test/xpu_api/numerics/complex.number/complex.transcendentals/pow_scalar_complex.pass.cpp

```diff
@@ -24,7 +24,7 @@ test(const T& a, const dpl::complex<T>& b, dpl::complex<T> x)
 {
     dpl::complex<T> c = dpl::pow(a, b);
     is_about(dpl::real(c), dpl::real(x));
-    assert(std::abs(dpl::imag(c)) < 1.e-6);
+    assert(std::abs(dpl::imag(c)) < 1.e-6f);
```

Comment:
I think should be `T(1.e-6)`

Reply:
Up to me - I think that both variants are correct.
Fixed.